docker/Dockerfile

```diff
@@ -0,0 +1,35 @@
+# FROM ubuntu:22.04 as buildenv
```

Comment:
I can revive this and build in a separate container, we just need 18.04 instead as that's what budtmo/docker-android is based on.